R/AllClasses2.R

```diff
@@ -207,6 +207,7 @@ setReplaceMethod(f="[[", signature="SYSargs2", definition=function(x, i, j, valu
 loadWorkflow <- function(targets=NULL, wf_file, input_file, dir_path=".") {
   if(!file.exists(file.path(dir_path, wf_file))==TRUE) stop("Provide valid '.cwl' file. Check the file PATH.")
   if(!file.exists(file.path(dir_path, input_file))==TRUE) stop("Provide valid 'files.'.yml' file. Check the file PATH.")
+  if(all(!is.null(targets) && !file.exists(targets)==TRUE)) stop("Provide valid 'targets' file. Check the file PATH.")
```

Comment:
Do you mean you want check if the variable `target` is empty and check if the path exists? I thought you want to use `any` : any of the conditions, then stop?

Reply:
@lz100 No, because we accept ```targets=NULL```. If ```targets``` is different then NULL, should be the path for the file ```targets.txt```. And we want to check if the path is correct, the path exists. If we use ```any``` it will create an error when ```targets=NULL```.